crates/rspack_core/src/compiler/compilation.rs

```diff
@@ -869,19 +869,33 @@ impl Compilation {
       .into_par_iter()
       .map(|job| {
         let module = job.module;
-        self
-          .old_cache
-          .code_generate_occasion
-          .use_cache(job, |module, runtime| {
-            let module = module_graph
-              .module_by_identifier(&module)
-              .expect("should have module");
-            module.code_generation(self, Some(runtime), None)
-          })
-          .map(|(res, runtimes, from_cache)| (module, res, runtimes, from_cache))
+        let runtimes = job.runtimes.clone();
```

Comment:
Can we avoid the clone here if we `map` the result of `use_cache`

Reply:
it needs clone anyway in the next step